datasets.py

```diff
@@ -187,6 +187,13 @@ def enumterms():
     return "/term/enum/", "2018-05-29T14:36:01.337Z", graph
 
 
+@compiler.dataset
+def aatterms():
+    graph = Graph().parse(str(compiler.path('source/aat-terms.ttl')), format='turtle')
```

Comment:
This file is missing? Also, we should probably not call this "aat", since we (probably) don't want to import all terms from AAT.

Reply:
Uuups... forgot to add it

Reply:
True. We will try to brainstorm something as comprehensible as possible. If you have any suggestions, we will be happy to accept them :)

Reply:
Worth mentioning is that since we do not yet know how many terms we will add, we have so far added them directly to a turtle file, instead of constructing them with sparql